test/protocol-tests/src/main/resources/codegen-resources/query/service-2.json

```diff
@@ -97,12 +97,6 @@
     "BlobType":{"type":"blob"},
     "Boolean":{"type":"boolean"},
     "Double":{"type":"double"},
-    "EmptyModeledException":{
```

Comment:
Why did this get removed?

Reply:
It was actually a duplicated entry: there's another EmptyModeledException below. Jackson-databind ignored it, but Jackson-jr required additional flags to support duplicate entries. Since it was invalid anyway, I just fixed it.